packages/medusa/src/repositories/money-amount.ts

```diff
@@ -328,10 +338,22 @@ export const MoneyAmountRepository = dataSource
         )
       }
 
-      const [prices, count] = await qb.getManyAndCount()
+      let prices
+      let count = 0
+
+      if (shouldCount) {
+        const results = await qb.getManyAndCount()
+        prices = results[0]
+        count = results[1]
```

Comment:
todo: omit count

Reply:
If shouldCount is true we should assign the count. I am not sure to follow

Reply:
haha sorry, this was an old comment - ignore 🤦